packages/editor/src/components/post-saved-state/index.js

```diff
@@ -66,10 +66,14 @@ export class PostSavedState extends Component {
 			} );
 
 			return (
-				<span className={ classes }>
-					<Dashicon icon="cloud" />
-					{ isAutosaving ? __( 'Autosaving' ) : __( 'Saving' ) }
-				</span>
+				<Animate type="loading">
```

Comment:
Seeing this component used and the fact that it adds a level in the hierarchy makes me wonder if we could replace it with a hook. (Not for this PR though)

```js
const  animationProps = useAnimation( type );
```

Reply:
Yeah, I agree — implementing this felt a bit verbose.